packages/e2e/test/wallet/SingleAddressWallet/mint.test.ts

```diff
@@ -25,7 +25,7 @@ describe('SingleAddressWallet/mint', () => {
     const genesis = await firstValueFrom(wallet.genesisParameters$);
 
     const aliceKeyAgent = await createStandaloneKeyAgent(
-      util.generateMnemonicWords(),
+      env.MINT_POLICY_SIGNER_MNEMONICS1.split(' '),
```

Comment:
Is there a reason we can't use accounts from the credentials provided in `KEY_MANAGEMENT_PARAMS`, for both Alice and Bob?